eclass/dist-kernel-utils.eclass

```diff
@@ -38,7 +40,19 @@ dist-kernel_build_initramfs() {
 	local rel_image_path=$(dist-kernel_get_image_path)
 	local image=${output%/*}/${rel_image_path##*/}
 
-	local args=(
+	local args=()
+	if command -v kernel-install &> /dev/null; then
+		# Mismatching the layout and dracut uefi setting just creates a mess
+		local layout="$(kernel-install inspect |  sed -n 's/^KERNEL_INSTALL_LAYOUT: //p' 2>&1)"
```

Comment:
Double space before sed, the quotes (`="$()"`) are also unneeded.

What's that `2>&1` for? You need to pickup sed error messages inside the layout variable?

Reply:
This is a remnant from some testing earlier that I forgot to remove